### ISSUE TITLE
Opt-in remote execution config does not work in 1.4.0

### ISSUE THREAD

**Issue by andrey-bolduzev** [skrugly/mirakle#58]
After upgrading to mirakle 1.4.0 from 1.3.6, the following config in `.gradle/init.d/mirakle_init.gradle` stopped working:

```groovy
initscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath "com.instamotor:mirakle:1.4.0"
    }
}

if (startParameter.taskNames.contains("mirakle")) {

    apply plugin: Mirakle

    rootProject {
        mirakle {
            host "remote_machine"
            fallback true
            rsyncToRemoteArgs += ["--stats", "-h", "--info=progress2", "-zz", "--compress-level=5"]
            rsyncFromRemoteArgs += ["--stats", "-h", "--info=progress2", "-zz", "--compress-level=5"]

            if (!project.hasProperty("dl")) {
                rsyncFromRemoteArgs += ["--dry-run"]
            }
        }
    }
}
```

In 1.4.0, any gradle command with the `mirakle` task attached prints the default gradle message, as if the actual task parameter got stripped away:
```
Starting a Gradle Daemon, 1 busy and 1 incompatible and 4 stopped Daemons could not be reused, use --status for details

> Configure project :
Here's Mirakle 1.4.0. All tasks will be executed on remote_machine.

> Task :help

Welcome to Gradle 6.7.

To run a build, run gradlew <task> ...

To see a list of available tasks, run gradlew tasks

To see a list of command-line options, run gradlew --help

To see more detail about a task, run gradlew help --task <task>

For troubleshooting, visit https://help.gradle.org
Total time : 0.919 secs
1 actionable task: 1 executed
```

Is there anything I should be doing differently in 1.4.0?

Thank you.

**Comment by skrugly** [comment]
Hey, thanks for reporting, I found and fixed the bug. 
Could you try `1.4.1-RC-1` and feel free to close the issue if it fixed after that.
I keep doing fixes for `1.4.1` version and will release it later.

**Comment by andrey-bolduzev** [comment]
Works perfectly now. Thank you for the quick fix!
